.github/workflows/build.yml

```diff
@@ -24,6 +24,10 @@ jobs:
       uses: actions/setup-dotnet@v1
       with:
         dotnet-version: ${{ matrix.dotnet }}
+    - name: Setup .NET 6
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: '6.x'
```

Comment:
@Numpsy why not upgrade from v5 to v6 everywhere? (specifically, am wondering why you left 5.0.x in L15)